### PR TITLE
Quiz and result page completed

### PR DESCRIPTION
**Pull request by animeshgarai09**
Quiz page : [https://deploy-preview-3--felix-quiz-dev.netlify.app/quiz.html](url)
Result page : [https://deploy-preview-3--felix-quiz-dev.netlify.app/result.html](url)

### CODE REVIEW

File: styles/results.scss [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            justify-content: space-between;
+            padding: 0.3rem;
+            span {
+                font-weight: 500;
+
+                &:first-child {
+                    font-size: 0.9rem;
+                    color: var(--gray-2);
+                }
+                // &:last-child {
```
Comment (keshavkumar-in) [inline styles/results.scss:41]: You can remove the commented code .

File: result.html [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </main>
+        <footer class="footer">
+            <div class="footer_wrapper">
+                <span>Made with <i class="fas fa-heart"></i> by <a href="https://portfolio-alpha-liart.vercel.app/" class="text-gray"> Animesh Garai</a></span>
+                <div class="footer__links">
+                    <a href="https://github.com/animeshgarai09" target="_blank"><i class="fab fa-github"></i></a>
```
Comment (keshavkumar-in) [inline result.html:164]: If you are using `target blank` you should also use `rel` attribute with `noopener noreferrer` value. It helps in securing your site from malicious attacks.

**Review by keshavkumar-in — APPROVED** [review]
Just has a few suggestions, otherwise good to merge.

**Review by shahbazibnafzal — APPROVED** [review]
Amazing piece of work Animesh. 
You might consider putting validation on the instruction page so that user can only go ahead once he agrees with the rules however that can be added later as well.

Great job man!

**Review by vashnavichauhan1825 — APPROVED** [review]
approved for merge.

**Comment by vashnavichauhan1825** [comment]
UI looks cool and awesome 🔥🔥 you rock

**Review by ParthParolekar — APPROVED** [review]
Looks good to me! The UI is amazing

**Review by arjun-computer-geek — APPROVED** [review]

File: result.html (hunk 2 above)
Comment (animeshgarai09) [reply result.html:164]: Thank you for the insight. will change 👍🏻

**Review by VLeads — APPROVED** [review]
Nice work done animesh
Code is clean and all good practices are been followed.
Also you have done such an awesome work on UI part.
Keep going !! 💯

File: result.html [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+                                    <input type="radio" name="options" id="third" disabled />
+                                    <label class="options" for="third"> To dry out wet soil </label>
+                                    <input type="radio" name="options" id="fourth" disabled />
+                                    <label class="options right" for="fourth"> To prevents bugs </label>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
```
Comment (abhishektanwar) [inline result.html:158]: It would be better to make the options here flex column or 2 column grid layout

image

File: styles/results.css [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+
+input[type=radio] {
+  display: none;
+}
+input[type=radio]:checked + .options {
+  background: var(--primary);
+  color: var(--white);
+}
+
+main {
```
Comment (abhishektanwar) [inline styles/results.css:76]: Please try to remain consistent with spaces between css classes.

File: styles/results.scss [hunk 5]
```diff
@@ -0,0 +1,90 @@
+@use "quiz";
+
+main {
+    display: flex;
+    gap: 2rem;
+    //   padding: 3rem 1rem;
```
Comment (abhishektanwar) [inline styles/results.scss:6]: Please remove commented code.

**Review by abhishektanwar — APPROVED** [review]
Clean and readable comment. Please check suggestion for the results page. Good to merge

File: instructions.html [hunk 6]
```diff
@@ -4,7 +4,7 @@
         <meta charset="UTF-8" />
         <meta http-equiv="X-UA-Compatible" content="IE=edge" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <title>Home | Felix Quiz</title>
+        <title>Instructions | Felix Quiz</title>
         <link rel="stylesheet" href="https://felix-ui-dev.netlify.app/components/main.css" />
         <!-- <link rel="stylesheet" href="/components/main.css" /> -->
```
Comment (arjun-computer-geek) [inline instructions.html:10]: It's bad idea to leave commented code

File: result.html [hunk 7]
```diff
@@ -0,0 +1,171 @@
...
+                    <span class="number-badge">4</span>
+                    <div class="quiz-container">
+                        <div class="quiz-wrapper">
+                            <div class="quiz-body">
+                                <p>Which of the following is NOT a benefit of using mulch?</p>
+
+                                <div class="quiz-options">
+                                    <input type="radio" name="options" id="first" disabled />
+                                    <label class="options" for="first"> To keep the soil cool </label>
+                                    <input type="radio" name="options" id="second" disabled />
```
Comment (arjun-computer-geek) [inline result.html:126]: that's nice to use for attribute in label tag.

**Review by arjun-computer-geek — COMMENTED** [review]
Cleaner code. Nice Ui. Best practice used in code. Easily to understand

**Pull request merged**